pages/api-reference/transfers.mdx

```diff
@@ -923,7 +923,7 @@ appToken.get(`${transferUrl}/failure`).then((res) => res.body.code); // => 'R01'
 
 ## Cancel a transfer
 
-When a bank transfer is eligible for cancellation, Dwolla returns a `cancel` link when [getting the transfer by Id](#retrieve-a-transfer). This cancel link is used to trigger the cancellation, preventing the bank transfer from processing further. A bank transfer is cancellable up until 4pm CT on that same business day if the transfer was initiated prior to 4PM CT. If a transfer was initiated after 4pm CT, it can be cancelled anytime before 4pm CT on the following business day.
+When a bank transfer is eligible for cancellation, Dwolla returns a `cancel` link when [getting the transfer by Id](#retrieve-a-transfer). This cancel link is used to trigger the cancellation, preventing the bank transfer from processing further. A bank transfer is cancellable up until 4pm CT on that same business day if the transfer was initiated prior to 4pm CT. If a transfer was initiated after 4pm CT, it can be cancelled before 4pm CT on the following business day. **Note:** This is not a hard cut off. Occassionally, the export out of the Dwolla Network may be delayed or could take some time to process. This may cause transfers created close after 4pm CT to be exported the same business day. It is recommended to rely on the `cancel` link returned on the transfer resource to determine if a transfer is cancellable rather than relying on the export timing.
```

Comment:
"This may cause transfers created close after 4pm"

This may cause transfers created close to after 4pm or This may cause transfer created just after 4pm

*close to
or
* just after

Reply:
Thanks Kelly! Should be fixed with this [commit](https://github.com/Dwolla/dev-portal/pull/156/commits/568353696a3a9db5eedc80fa30bdfb793b773fb8). 👍